Script/PRONTO.py

```diff
@@ -785,19 +920,23 @@ def remisse_mail_writer(remisse_file,ipd_no,ipd_consent,DNA_normal_sampleID,RNA_
 	text3 = pg2.add_run("diagnose")
 	text3.font.color.rgb = docRGBColor(0,176,80)
 	pg2.add_run(if_normal_sampleID)
-	if(TMB_DRUP >= 0 and TMB_DRUP <= 5):
-		TMB_position = "lav"
-	if(TMB_DRUP > 5 and TMB_DRUP <= 20):
-		TMB_position = "intermediær"
-	if(TMB_DRUP > 20):
-		TMB_position = "høy"
+	if(str_TMB_DRUP == "NA"):
+		TMB_string = "Upålitelig, ikke beregnet\n"
+	else:
+		if(TMB_DRUP >= 0 and TMB_DRUP <= 5):
+			TMB_position = "lav"
+		if(TMB_DRUP > 5 and TMB_DRUP <= 20):
+			TMB_position = "intermediær"
+		if(TMB_DRUP > 20):
+			TMB_position = "høy"
+		TMB_string = str(TMB_DRUP) + " mut/Mb; " + TMB_position + "\n"
```

Comment:
refactor into a separate function that takes TMB_DRUP and returns the generated string. Would be a nice candidate for starting to add unit testing to the project.

Reply:
Sorry for the late replying. This is the function to generate the email draft file which is only used by OUS I think. This is the content changes for some special samples requested from the molecular biologist team of OUS. Not sure what do you mean by "unit testing". Do you want a testing sample for it?

Reply:
Thanks. I will take a check. :)

Reply:
An issue is created now.